src/common/Requests.ts

```diff
@@ -150,7 +150,7 @@ class _Requests {
 		if (!Shared.storage.options.grantCookies || !browser.cookies || !browser.webRequest) {
 			return;
 		}
-		const domainMatches = /https?:\/\/(?:www\.)?(?:.+?)(?<domain>\/.*)?$/.exec(request.url);
+		const domainMatches = /https?:\/\/(?:www\.)?(?<domain>.+?)(?:\/.*)?$/.exec(request.url);
```

Comment:
Thanks! I actually found and fixed this issue as well on my local, but haven't had time to push it yet.